src/test/java/com/wikia/webdriver/common/contentpatterns/AdsFandomContent.java

```diff
@@ -20,6 +23,9 @@
     SLOT_SELECTORS.put(TOP_BOXAD, "div[id$='TOP_BOXAD_0__container__']");
     SLOT_SELECTORS.put(INCONTENT_BOXAD, "div[id$='INCONTENT_BOXAD_0__container__']");
     SLOT_SELECTORS.put(BOTTOM_BOXAD, "div[id$='BOTTOM_BOXAD_0__container__']");
+    SLOT_SELECTORS.put(GPT_TOP_LEADERBOARD, "#gpt-top-leaderboard");
+    SLOT_SELECTORS.put(GPT_BOTTOM_LEADERBOARD, "#gpt-bottom-leaderboard-desktop");
+    SLOT_SELECTORS.put(GPT_BOTTOM_LEADERBOARD_MOBILE, "#gpt-bottom-leaderboard-mobile");
```

Comment:
Do we really need those new elements? Doesn't it work with existing selectors:
* `div[id$='TOP_LEADERBOARD_0__container__']` for top leaderboards (any of them, including UAP and mobile),
* `div[id$='BOTTOM_LEADERBOARD_0__container__']` for bottom leaderboards.